src/integrations/windowsIntegrationManager.ts

```diff
@@ -420,13 +455,28 @@ export default class WindowsIntegrationManager implements IntegrationManager {
         },
         executable, 'kubeconfig', '--show');
 
+      console.log(`WSL distro "${ distro.name }: wsl-helper output: "${ stdout }"`);
       if (['true', 'false'].includes(stdout.trim())) {
         return stdout.trim() === 'true';
       } else {
-        return stdout.trim();
+        return `Error: ${ stdout.trim() }`;
       }
     } catch (error) {
-      return (typeof error === 'object' && error?.toString()) || false;
+      if (typeof error === 'object' && error) {
+        const errorString = error.toString();
+
+        console.log(`WSL distro "${ distro.name }: error: ${ errorString }`);
+
+        return `Error: ${ errorString }`;
+      } else if (typeof error === 'string') {
+        console.log(`WSL distro "${ distro.name }: error: ${ error }`);
+
+        return `Error: ${ error }`;
+      } else {
+        console.log(`WSL distro "${ distro.name }: error: ${ error }`);
```

Comment:
At this point, you can move the logging out of the check (since all three log the same thing).

Reply:
The cases are different enough that I am having a hard time figuring out a way to improve it beyond what is already there. Maybe the below will illustrate it:

error of type `object` -> log error.toString(), return error.toString()
error of type `string` -> log error, return error
error of other type -> log error, return "unexpected error getting state of distro"

There doesn't seem to be a way of unifying the three cases...

Reply:
All three cases logging the same message, just how the error is printed is different:
- `object`: `error.toString()` (note that the backticks implicitly call `toString()`)
- `string`: `error`, which is the same as `error.toString()`
- other: `error`, which is the same as `error.toString()`

I'm _not_ advocating merging the return value, just the logging.

Reply:
Ah I see. Done.